custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -309,7 +311,7 @@ def _get_size_and_units(size):
     def _wait_for_server_to_start(self):
         while True:
             try:
-                response = requests.get(self._url_server_address)
+                response = requests.get(self._url_server_address, timeout=1)
```

Comment:
what if 1s is not enough, and will give false positives for already running server?

Reply:
I think we should use `_timeout` instead of 1s -- in does't matter if the request timed out during a single request or after several requests
```suggestion
                response = requests.get(self._url_server_address, timeout=_timeout)
```

Reply:
> we need to remove the while: True loop

we remove the loop?